src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -49,6 +49,11 @@ private DynValue Processing_Loop(int instructionPtr, bool canAwait = false)
 
 					++executedInstructions;
 
+					if (m_Script.Options.InstructionLimit > 0 && executedInstructions > m_Script.Options.InstructionLimit)
+					{
+						throw new ScriptRuntimeException($"Maximum limit of {m_Script.Options.InstructionLimit} exceed. Script terminated.");
```

Comment:
Grammar: exceed -> exceeded

Reply:
Fixed in 0f19ff28b9ce3fe1903549f2d9ab63a12c36316c